grafana-plugin/src/pages/incidents/Incidents.tsx

```diff
@@ -583,6 +587,40 @@ class Incidents extends React.Component<IncidentsPageProps, IncidentsPageState>
     );
   }
 
+  renderLabels(item: AlertType) {
+    if (!item.labels.length) {
+      return null;
+    }
+
+    return (
+      <TooltipBadge
+        tooltipTitle=""
+        borderType="secondary"
+        icon="tag-alt"
+        addPadding
+        text={item.labels?.length}
+        tooltipContent={
+          <VerticalGroup spacing="sm">
+            {item.labels?.length
+              ? item.labels.map((label) => (
+                  <HorizontalGroup spacing="sm" key={label.key.id}>
+                    <LabelTag label={label.key.name} value={label.value.name} key={label.key.id} />
+                    <Button
+                      size="sm"
+                      icon="filter"
+                      tooltip="Apply filter"
+                      variant="secondary"
+                      onClick={this.getApplyLabelFilterClickHandler(label)}
+                    />
+                  </HorizontalGroup>
+                ))
+              : 'No labels attached'}
```

Comment:
Same here, in line 591 we're already checking `item.labels.length` and we're returning `null` if it's falsy. So second check is redundant I believe